lib/miq_automation_engine/engine/miq_ae_service.rb

```diff
@@ -185,6 +185,10 @@ def root
       @root_object ||= object("/")
     end
 
+    def categories
+      MiqAeMethodService::MiqAeServiceModelBase.wrap_results(Classification.categories)
+    end
```

Comment:
@mkanoor I would prefer we move this logic into the `classification` service model and expose it as a class method there.  I think the call would end up looking something like: `$evm.vmdb('classification').categories`.